airflow/utils/log/logging_mixin.py

```diff
@@ -92,7 +92,7 @@ def write(self, message):
             self._buffer += message
         else:
             self._buffer += message
-            self.logger.log(self.level, self._buffer)
+            self.logger.log(self.level, self._buffer.rstrip('\n'))
```

Comment:
@r39132 I think you will get this `\n` for free, therefore I think this is a good change. Wouldn't `.rstrip()` be sufficient, to also trim all kinds of whitespace at the end.
```
MacBook-Pro-van-Fokko:~ fokkodriesprong$ python3
Python 3.7.0 (default, Jun 29 2018, 20:13:13) 
[Clang 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 'hoi\n'
'hoi\n'
>>> 'hoi\n'.rstrip()
'hoi'
```

Reply:
Got it.

Reply:
Sounds reasonable. I assuming the log call at that line will introduce a single `\n` after rstrip() removes all of the trailing white space in the buffer. @ckljohn Can you make this change? Then we should be good to merge it.